core/orchestrator.go

```diff
@@ -753,14 +753,33 @@ func (rtm *RemoteTranscoderManager) Manage(stream net.Transcoder_RegisterTransco
 	rtm.liveTranscoders[transcoder.stream] = transcoder
 	rtm.remoteTranscoders = append(rtm.remoteTranscoders, transcoder)
 	sort.Sort(byLoadFactor(rtm.remoteTranscoders))
+	var totalLoad, totalCapacity, liveTranscodersNum int
+	if monitor.Enabled {
+		liveTranscodersNum = len(rtm.liveTranscoders)
+		totalLoad, totalCapacity = rtm.totalLoadAndCapacity()
+	}
 	rtm.RTmutex.Unlock()
+	if monitor.Enabled {
+		monitor.SetTranscodersNumber(int64(liveTranscodersNum))
+		monitor.SetTranscodersCapacity(int64(totalCapacity))
+		monitor.SetTranscodersLoad(int64(totalLoad))
```

Comment:
This is starting to feel verbose - would it make sense to have a single function in `monitor` that takes all 3 parameters?

Reply:
fixed